internal/pkg/agent/application/gateway/fleet/fleet_gateway.go

```diff
@@ -175,28 +161,12 @@ func (f *fleetGateway) worker() {
 			if err != nil {
 				continue
 			}
-
-			actions := f.queueScheduledActions(resp.Actions)
-			actions, err = f.dispatchCancelActions(actions)
-			if err != nil {
-				f.log.Error(err.Error())
+			actions := make([]fleetapi.Action, len(resp.Actions))
+			for idx, a := range resp.Actions {
+				actions[idx] = a
```

Comment:
Instead of this forloop copy should work.

```
copy(actions, resp.Actions)
```

Reply:
ok, loop was the original implementation. i'll change to copy

Reply:
I removed the change as the build was failing